src/java/org/apache/cassandra/metrics/ChunkCacheMetrics.java

```diff
@@ -35,7 +35,12 @@ public interface ChunkCacheMetrics extends StatsCounter, CacheMetrics
 {
     static ChunkCacheMetrics create(ChunkCache cache)
     {
-        return USE_MICROMETER.getBoolean() ? new MicrometerChunkCacheMetrics(cache) : new CodahaleChunkCacheMetrics(cache);
+        return create(cache, "chunk_cache");
+    }
+
+    static ChunkCacheMetrics create(ChunkCache cache, String prefix)
+    {
+        return USE_MICROMETER.getBoolean() ? new MicrometerChunkCacheMetrics(cache, prefix) : new CodahaleChunkCacheMetrics(cache);
```

Comment:
Supernit: I'd split the long line.
```suggestion
        return USE_MICROMETER.getBoolean()
               ? new MicrometerChunkCacheMetrics(cache, prefix)
               : new CodahaleChunkCacheMetrics(cache);
```